modules/articles/client/services/models/article.client.model.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
I'd say if Angular calls it a service, so should we. Let's not change it just because it matches server-side wording or whatever else cosmetic reason we may have.